android/src/main/kotlin/com/signify/hue/flutterreactiveble/channelhandlers/CharNotificationHandler.kt

```diff
@@ -30,9 +30,11 @@ class CharNotificationHandler(private val bleClient: com.signify.hue.flutterreac
                 .uuidFromByteArray(request.characteristic.characteristicUuid.data.toByteArray())
         val subscription = bleClient.setupNotification(request.characteristic.deviceId, charUuid)
                 .observeOn(AndroidSchedulers.mainThread())
-                .subscribe { value ->
+                .subscribe({ value ->
                     handleNotificationValue(request.characteristic, value)
-                }
+                }, {
```

Comment:
@werediver  previously we just printed it to the console but since we removed it in last version this came to the surface. I am actually thinking that a lot of issues regarding to notifications that were previously reported on iOS may also be reproducible on Android but since we never propagated an error before it never came to the surface. I will check later for other areas where we may have this issue